packages/cli/package.json

```diff
@@ -29,9 +29,12 @@
     "access": "public"
   },
   "files": [
-    "dist"
+    "dist",
+    "patches"
   ],
   "dependencies": {
-    "dedent": "1.6.0"
+    "dedent": "1.6.0",
+    "execa": "^9.5.3",
+    "patch-package": "^8.0.0"
```

Comment:
should be pin these deps?

Reply:
In this case I would see flexibility over "predictability" since they're currently only required for the patch.